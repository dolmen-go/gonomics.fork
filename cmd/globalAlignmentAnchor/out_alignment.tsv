chr1	1	10	species1_gap	chr1	20	30	species2_gap	-858	[{9 0} {1 1}]
chr2	30	40	species1_gap	chr2	10	18	species2_gap	-595	[{5 0} {2 2} {3 0}]
