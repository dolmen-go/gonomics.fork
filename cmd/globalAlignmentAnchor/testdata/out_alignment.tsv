chr1	1	10	species1_gap	chr1	1	15	species2_gap	-573	[{1 0} {1 1} {8 0} {4 1}]
chr1	20	30	species1_gap	chr1	20	30	species2_gap	-516	[{10 0}]
chr1	40	700	species1_gap	chr1	35	1300	species2_gap	-23978	[{21 0} {129 1} {5 0} {3 1} {9 0} {17 1} {23 0} {4 1} {3 0} {42 1} {15 0} {31 1} {4 0} {21 1} {19 0} {10 1} {14 0} {22 1} {17 0} {8 1} {19 0} {11 1} {9 0} {4 2} {15 0} {20 1} {12 0} {19 1} {8 0} {9 1} {26 0} {3 2} {38 0} {9 1} {13 0} {19 1} {5 0} {27 1} {8 0} {8 1} {7 0} {27 1} {7 0} {12 1} {16 0} {10 1} {7 0} {41 1} {9 0} {5 1} {24 0} {3 1} {23 0} {5 1} {25 0} {25 1} {24 0} {1 1} {16 0} {36 1} {25 0} {1 2} {24 0} {8 1} {9 0} {9 1} {12 0} {1 1} {6 0} {8 1} {14 0} {2 2} {17 0} {4 1} {19 0} {6 2} {40 0} {13 1} {16 0} {1 2} {13 0} {5 1} {7 0}]
chr2	1	5	species1_gap	chr2	1	8	species2_gap	-547	[{4 0} {3 1}]
chr2	35	550	species1_gap	chr2	20	850	species2_gap	39850	[{1 1} {515 0}]
