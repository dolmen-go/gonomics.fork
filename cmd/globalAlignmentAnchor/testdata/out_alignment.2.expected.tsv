chr1	20	30	species1_gap	chr1	20	30	species2_gap	-2038	[{10 0}]
chr2	1	5	species1_gap	chr2	1	8	species2_gap	-1806	[{1 0} {3 1} {3 0}]
